### PR TITLE
Filesystem check

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
This pull request will add:
- ability to write filesystem checks
- check for help file presence

----

Also corrected typo in the filename for `io_k8s_display-name` check.

### CODE REVIEW

**Comment by lachmanfrantisek** [comment]
Example of the result:

```json
"best_practices": [
        {
            "name": "help_file_required",
            "ok": false,
            "status": "failed",
            "description": "Just like traditional packages, containers need some 'man page' information about how they are to be used, configured, and integrated into a larger stack.",
            "message": "The 'helpfile' has to be provided.",
            "reference_url": "https://fedoraproject.org/wiki/Container:Guidelines#Help_File",
            "severity": "required",
            "logs": [
                "File '/help.1.' is not present.",
                "File '/README.md' is not present."
            ]
        }
    ]
```

File: colin/checks/best_practices/help_file.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+class HelpFileCheck(FileSystemCheck):
+
+    def __init__(self):
+        super().__init__(name="help_file_required",
+                         message="The 'helpfile' has to be provided.",
+                         description="Just like traditional packages, containers need "
+                                     "some 'man page' information about how they are to be used,"
+                                     " configured, and integrated into a larger stack.",
+                         reference_url="https://fedoraproject.org/wiki/Container:Guidelines#Help_File",
+                         files=['/help.1.', '/README.md'],
```
Comment (phracek) [inline colin/checks/best_practices/help_file.py]: Why `/help.1.`? It should be `/help.1`, right?

Comment (lachmanfrantisek) [reply colin/checks/best_practices/help_file.py]: Thanks!

File: colin/checks/result.py [hunk 2]
```diff
@@ -39,20 +40,6 @@ def __init__(self, ok, description, message, reference_url, check_name, severity
         self.correction_diff = correction_diff
 
 
-class ContainerCheckResult(AbstractResult):
```
Comment (phracek) [inline colin/checks/result.py]: Why do you remove it? Just a question. Would you like to have only one `CheckResult`?

Comment (lachmanfrantisek) [reply colin/checks/result.py]: The `LabelCheck` implementation is same for containers and images and I need to return the same instance for both types.

(For now, the `ContainerCheckResult` x `ImageCheckResult` gives us no value.)

File: colin/checks/best_practices/help_file.py (hunk 1 above)
Comment (lachmanfrantisek) [reply colin/checks/best_practices/help_file.py]: I've discovered, that Fedora requires one of the `help.1`/`README.md`, RHEL specifies only `help.1`.

I've created two different checks for each of them.

**Comment by phracek** [comment]
@lachmanfrantisek Why do we have two checks, like `help_file.py` and `help_file_or_readme.py`?
On fedora.json and rhel.json you have only help_file. help_file_or_readme is missinh. Any reason?

**Pull request merged**